lib/SimplePie/SimplePie/Content/Type/Sniffer.php

```diff
@@ -121,9 +123,7 @@ public function get_type()
 
 				return $official;
 			}
-			elseif ($official === 'text/html'
-				|| $official === 'text/xml'	//FreshRSS
```

Comment:
@candrews 

> Is there anything else that I can / should do to help this along?

I don't know all the specifics but any line with a comment like this presumably needs manual attention.

Reply:
Thanks for pointing this out!